lib/Backend/BackendUtils.cpp

```diff
@@ -37,17 +37,40 @@ static llvm::cl::opt<bool> reuseActivationsMemory(
     llvm::cl::desc("Should activation memory allocations be reused"),
     llvm::cl::init(true), llvm::cl::cat(BackendUtilsCat));
 
+glow::runtime::RuntimeBundle::RuntimeBundle(
+    glow::runtime::RuntimeBundle &&rhs) {
+  *this = std::move(rhs);
+}
+
+glow::runtime::RuntimeBundle &glow::runtime::RuntimeBundle::
+operator=(glow::runtime::RuntimeBundle &&rhs) {
```

Comment:
early exit for self assignment, otherwise we made an object non valid `rhs.isValid_ = false;`.